src/main/java/org/testeditor/fixture/host/util/LineReader.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.host.util;
+
+import org.testeditor.fixture.host.locator.Locator;
+import org.testeditor.fixture.host.locator.LocatorByStartStop;
+import org.testeditor.fixture.host.locator.LocatorByWidth;
+
+import java.util.List;
+
+/**
+ * This is a utility class for reading multiple lines on a Mainframe Screen. For
+ * reading multiple lines, we need a start point and an end point, these start
+ * and end points can received through a start row and start column and end row
+ * and end column.
+ *
+ */
+public class LineReader {
+
+    /**
+     * 
+     * @param lines
+     *            the rows which should be read.
+     * @param locator
+     *            see {@link Locator} the start and endpoint representation of
+     *            String to be read.
+     * @return the excerpted String from start to endpoint
+     */
+    public String readMultilines(List<String> lines, Locator locatorIn) {
+        StringBuffer sb = new StringBuffer();
+
+        LocatorByStartStop locator = (LocatorByStartStop) locatorIn;
+
+        int startRow = locator.getStartRow();
+        int endRow = locator.getEndRow();
+
+        for (int i = startRow - 1; i < endRow; i++) {
+            String line = lines.get(i).replace("data:  ", "");
+            if (i == startRow - 1) {
+                sb.append(line.substring(locator.getStartColumn() - 1));
+            } else if (i == endRow - 1) {
+                sb.append(line.substring(0, locator.getEndColumn()));
+            } else {
+                sb.append(line);
```

Comment:
are multiline fields always over the whole line? or are they rectangular?
'''''''+------+
+--+........|
|........+--+       
+------+

vs.

   +----+
    |.....|
    |.....|
   +----+